projects/components/src/metric-card/metric-card.component.ts

```diff
@@ -21,10 +25,22 @@ import { MetricCardIconPosition, MetricCardIndicatorType } from './metric-card';
             </ng-container>
           </ng-container>
         </div>
-        <div class="title-text">{{ this.titleText }}</div>
+        <div
+          [ngClass]="{
+            'large-title-text': this.cardSize === '${MetricCardSize.Large}',
+            'title-text': this.cardSize === '${MetricCardSize.Small}'
```

Comment:
instead of doing this, please do this.
```html
<div class="title-text" [ngClass]="this.cardSize">{{ this.titleText }}</div>
```
```css
.title-text {
  &.large {
      @include body-1-medium($gray-7);
      text-transform: capitalize;
    }
}
```

similarly below as well

Reply:
oh, got it, updated